Helper/PaymentMethods.php

```diff
@@ -487,4 +490,65 @@ public function isWalletPaymentMethod($notificationPaymentMethod): bool
         ];
         return in_array($notificationPaymentMethod, $walletPaymentMethods);
     }
+
+    /**
+     * Check if the method of the passed payment is equal to the method passed in this function
+     *
+     * @param $payment
+     * @param string $method
+     * @return bool
+     */
+    public function checkPaymentMethod($payment, string $method): bool
+    {
+        return $payment->getMethod() === $method;
+    }
+
+    /**
+     * Check if the passed payment method supports recurring functionality.
+     * If a payment method is in this list it does not necessarily imply that it is already supported
+     * by the plugin.
+     *
+     * Currently only SEPA is allowed on our Magento plugin.
+     *
+     * @param string $paymentMethod
+     * @return bool
+     */
+    public function paymentMethodSupportsRecurring(string $paymentMethod): bool
+    {
+        $paymentMethodRecurring = [
+            /*'ach',
+            'amazonpay',
+            'applepay',
+            'directdebit_GB',
+            'bcmc',
+            'dana',
+            'dankort',
+            'eps',
+            'gcash',
+            'giropay',
+            'googlepay',
+            'paywithgoogle',
+            'gopay_wallet',
+            'ideal',
+            'kakaopay',
+            'klarna',
+            'klarna_account',
+            'klarna_b2b',
+            'klarna_paynow',
+            'momo_wallet',
+            'paymaya_wallet',
+            'paypal',
+            'trustly',
+            'twint',
+            'uatp',
+            'billdesk_upi',
+            'payu_IN_upi',
+            'vipps',
+            'yandex_money',
+            'zip',*/
```

Comment:
The intention for this comment is to serve as a guide for PM codes that could be enabled for recurring in customizations, right?
If so, would it be better to move it to the function's doc block?

Reply:
Updated in [8717edd](https://github.com/Adyen/adyen-magento2/pull/1389/commits/8717edd3b1595cae4346274f1449a671c3168516)